llvm/lib/Target/AArch64/AArch64ExpandPseudoInsts.cpp

```diff
@@ -838,6 +838,13 @@ bool AArch64ExpandPseudo::expandCALL_BTI(MachineBasicBlock &MBB,
   Call->addOperand(CallTarget);
   Call->setCFIType(*MBB.getParent(), MI.getCFIType());
 
```

Comment:
maybe we could just say:
`Call->copyImplicitOps(*MBB.getParent(), MI);`

Reply:
Much cleaner, thanks!